src/Page.js

```diff
@@ -799,8 +799,8 @@ Page.prototype.generate = function (builtFiles) {
       .then(result => addContentWrapper(result))
       .then(result => this.preRender(result))
       .then(result => this.insertSiteNav((result)))
-      .then(result => this.insertTemporaryStyles(result))
       .then(result => this.insertHeaderFile(result))
+      .then(result => this.insertTemporaryStyles(result))
```

Comment:
Should put this after insertHeaderFile? So that the header and footer are logically grouped together.

Reply:
Good suggestion, since there is a chance that `navbars` or `dropdown menus` are placed in a footer by the author.